R/auditCommonFunctions.R

```diff
@@ -2267,7 +2288,8 @@
     message <- gettextf("The sample is drawn with <i>seed %1$s</i>.",
                         options[["seed"]])
   } else {
-    message <- gettext("The first unit from every interval is selected.")
+    message <- gettextf("Unit %1$s is selected from each interval.",
```

Comment:
For a single argument it isnt necessary to give it a position, but it cant hurt either